app/controllers/admin/base_controller.rb

```diff
@@ -2,5 +2,10 @@ class Admin::BaseController < ApplicationController
   before_action :check_admin
 
   private
-  def check_admin; end
+  def check_admin
+    return if is_admin?
```

Comment:
Chị nghĩ ko cần method này đâu, trực tiếp sử dụng ở đây đi em 
```suggestion
    return if @current_user.admin?
```